frontend/public/components/container.jsx

```diff
@@ -251,21 +251,21 @@ const Details = (props) => {
 };
 
 export const ContainersDetailsPage = (props) => <div>
-  <PageHeading
-    detail={true}
-    title={props.match.params.name}
-    kind="Container"
-    breadcrumbsFor={() => [
-      {name: props.match.params.podName, path: `${props.match.url.split('/').filter((v, i) => i <= props.match.path.split('/').indexOf(':podName')).join('/')}`},
-      {name: 'Container Details', path: `${props.match.url}`},
-    ]} />
   <Firehose resources={[{
     name: props.match.params.podName,
     namespace: props.match.params.ns,
     kind: 'Pod',
     isList: false,
     prop: 'obj',
   }]}>
+    <PageHeading
+      detail={true}
+      title={props.match.params.name}
+      kind="Container"
+      breadcrumbsFor={() => [
+        {name: props.match.params.podName, path: `${props.match.url.split('/').filter((v, i) => i <= props.match.path.split('/').indexOf(':podName')).join('/')}`},
```

Comment:
I'd rather avoid path math and just build the URL for a pod.

Reply:
Now using `resourcePath(..)`

Reply:
Yeah I think this is cleaner, thanks. (I realize you had just moved the existing code previously.)